docs/copy-assets.md

```diff
@@ -13,7 +13,7 @@ You can copy these assets by using the `assets` option.
   "ngPackage": {
     "assets": [
       "CHANGELOG.md",
-      "./styles/**/*.theme.scss"
+      { "input": "src/styles", "glob": "**/*.scss", "output": "styles" }
```

Comment:
Can you leave an example for the simple glob please?

Reply:
updated